benefits/core/templates/core/includes/form.html

```diff
@@ -47,10 +50,11 @@
         $("#{{ form.id }}").on("submit", function(e) {
           if ("{{ form.submitting_value }}" !== "") {
             $("button[type=submit]", this)
+              .removeClass("spinner-hidden")
+              .addClass("disabled")
               .attr("role", "status")
```

Comment:
I wonder if this is necessary if `<span class="spinner-border spinner-border-sm" role="status"></span>` already has `role="status"`? I was not sure.

Reply:
After reading this https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/status_role, I don't think we should have two elements with `role=status` b/c it would announce the same status twice.

Reply:
Fixed in [`b0343d6`](https://github.com/cal-itp/benefits/pull/1088/commits/b0343d6b1f37ce26b5f107ae531219f16dd3d938)